test_bls.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+set -e
+set -x
+
+# rm -rf output
+
+# Single test
+RUST_LOG=info cargo run --release  -- -k 20 --function zkmain --output ./output --wasm ctest/bls.wasm setup
+
+RUST_LOG=info cargo run --release  -- -k 20 --function zkmain --output ./output --wasm ctest/bls.wasm single-prove
+RUST_LOG=info cargo run --release  -- -k 20 --function zkmain --output ./output --wasm ctest/bls.wasm single-verify --proof output/zkwasm.0.transcript.data
```

Comment:
remove `proof`, remove `aggregate-prove` and `aggregate-verify`

Reply:
done

Reply:
You haven't deleted them